bbn-1/pops/README.md

```diff
@@ -0,0 +1,422 @@
+- `--passphrase` specifies the password used to encrypt the key. This is
+  an optional flag used in the case of an encrypted keyring (recommended).
+- `--hd-path` is the HD derivation path of the private key.
+- `--keyring-backend` specifies the keyring backend. Any
+  of `[file, os, kwallet, test, pass, memory]` are available. You can find
+  more details about the available keyring backends
+  [here](https://docs.cosmos.network/v0.46/run-node/keyring.html#available-backends-for-the-keyring).
+  By default, the test keyring backend is used, although it is recommended
+  that you use an encrypted keyring.
+- `--recover` indicates whether the user wants to provide a seed phrase to
+  recover an existing key instead of creating a new one. This flag should only
+  be used if the user wants to import an existing keyring or seed phrase.
+
+```bash
+$~ babylond keys add my-baby-key-name --home /path/to/babylon/home/ --keyring-backend file
+Enter keyring passphrase (attempt 1/3):
+...
+- address: bbn17ew0he6svxrqj2c7mef7qsyg0assc2upa5gy7w
+  name: my-baby-key-name
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A5UjwURAlm0ndhMLfmNKIe4wzc8NnsBkNOgr7Ogx6kQQ"}'
+  type: local
+**Important** write this mnemonic phrase in a safe place.
+It is the only way to recover your account if you ever forget your password.
+bad mnemonic private split escape wage suit banner affair home fine rent erosion hurry bicycle throw tilt warfare silk rude real frown cabin few
+```
+
+> Reminder: if the `--keyring-backend` flag was used to create the key,
+> the same flag should be used later for accessing this key. At the end
+> of these steps, the Babylon key pair will be generated. The key pair
+> and the mnemonic generated must be stored in a safe place.
+> 
+> **⚠ Warning!**
+> 
+> Store the **mnemonic** in a safe place. The mnemonic is the only way
+> you can recover your keys in the case of loss or file system corruption.
+> 
+
+### 1.3. Import existing wallet
+
+If you wish to import an existing wallet with your mnemonic seed phrase,
+you can use the below command. Note that not all of the params below are
+required, please see the parameters list below to see which are optional.
+
+`babylond keys add <wallet-name> --recover --hd-path <hd-path> --home <home> --keyring-backend <backend>`
+
+Now you will be prompted for your `bip39 mnemonic` and you can enter
+your `Enter your 24-word seed phrase`.
+
+The output of the above command will be similar to the following:
+
+```bash
+- address: bbn1gsapezku653vxxmxvurh73s5j8lvxh6vzc6jjq
+  name: mykey
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AyFqOcOQeDTYpJctaWiQZ6SuM5y1a3batlsx6l2dgBAw"}'
+  type: local
+```
+
+## 2. Generate PoP for finality providers
+
+⚠️ This document assumes that the finality provider has been created and
+registered for phase-1 following the instructions in the
+[official documentation](https://github.com/babylonlabs-io/networks/tree/main/bbn-1/finality-providers).
```

Comment:
should we use a commit hash in the links?

Reply:
Hmmm I prefer the main branch here in case we might update the doc